index.html

```diff
@@ -0,0 +1,221 @@
+                        </ul>
+
+                    </div>
+                    <div class="body-context-button">
+                        <button type="button">
+                            See Project
+                        </button>
+                    </div>
+
+                </div>
+            </div>
+            <div class="body-context-projects">
+                <div class="context-projects-1">
+                    <button>See Project</button>
+                </div>
+                <div class="context-projects-2">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-3">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-2">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-3">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+                <div class="context-projects-3">
+                    <span>Data Dashboard Healthcare</span>
+                    <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                        industry's standard</p>
+                    <ul>
+                        <li>Html</li>
+                        <li>Boostrap</li>
+                        <li>Ruby</li>
+                    </ul>
+                </div>
+            </div>
+
+        </section>
+        <section class="section-about">
```

Comment:
Please align the image with the text in the About section. Kindly see the screenshot below.
![About](https://user-images.githubusercontent.com/35504149/135471451-0b5e0763-77d1-44f7-813e-2c8227d630f8.png)